courses/cunix/ex03/src/my_strcpy.c

```diff
@@ -0,0 +1,12 @@
+char *my_strcpy(char *dest, const char *src){
+	unsigned int count=0;
+	/*if(*src=='\0')
+		*dest='\0';
+	else*/
```

Comment:
in the future please remove comments that are not useful. Git remember everything and you always can search in git history commits for some previous parts of the code

Reply:
1.5 points